V1Documentation.md

```diff
@@ -1,53 +1,52 @@
-# Raygun4js
+# Raygun4js V1 Documentation
 
-[Raygun.io][raygunio] provider for client-side JavaScript
+The following documentation is for Raygun4js V1 only. Raygun4js V1 should only be used by legacy customers or for applications that block scripts.
```

Comment:
should be "for applications that can't use asynchronous loading, such as mobile apps in webviews (Cordova etc)"